infra/bicep/core/ai/cognitive-services.bicep

```diff
@@ -112,6 +65,7 @@ resource deployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01
 
 module privateEndpoint '../networking/private-endpoint.bicep' = if (empty(existing_CogServices_Name) && !empty(privateEndpointSubnetId)) {
   name: '${name}-private-endpoint'
+  dependsOn: deployment
```

Comment:
Had to have this, otherwise the PE was starting before the deployments were done and it was crashing...